kong/information_service.go

```diff
@@ -0,0 +1,48 @@
+package kong
+
+import (
+	"context"
+	"encoding/json"
+)
+
+// AbstractInfoService handles Kong's Information.
+type AbstractInfoService interface {
+	// Get retrieves the information concerning Kong
+	Get(ctx context.Context) (*Info, error)
+}
+
+type InfoService service
+
+// IsInMemory check if Kong is in memory
+func (r *RuntimeConfiguration) IsInMemory() bool {
+	return "off" == r.Database
+}
+
+// IsRBACEnabled check if RBAC are enabled
+func (r *RuntimeConfiguration) IsRBACEnabled() bool {
+	return "on" == r.RBAC
+}
+
+// Convert convert a object to another through json marshalling
+// unmarshalling
+func Convert(from, to interface{}) error {
```

Comment:
Do we really need this to be part of the public API?

Reply:
You're right - my comment https://github.com/Kong/go-kong/pull/65#discussion_r636800479 was misleading in this bit.

@mmorel-35 please unexport this function by renaming it to `convert`, and sorry about the confusion.